smsdk/client.py

```diff
@@ -154,13 +163,17 @@ def get_data_v1(self, ename, util_name, normalize=True, *args, **kwargs):
             # dict params strictly follow {'key':'value'} format
 
             # sub_kwargs = kwargs
-            if util_name in ['get_cycles', 'get_downtime', 'get_parts']:
+            if util_name in ['get_cycles', 'get_downtime', 'get_parts', 'get_factories', 'get_machines', 'get_machine_types']:
                 sub_kwargs = [kwargs]
             else:
                 sub_kwargs = self.fix_only(kwargs)
 
             if len(sub_kwargs) == 1:
-                data = dict_to_df(getattr(cls, util_name)(*args, **sub_kwargs[0]), normalize)
+                if util_name in ['get_factories', 'get_machines', 'get_machine_types']:
+                    # data = dict_to_df(getattr(cls, util_name)(*args, **sub_kwargs[0]), normalize)
+                    return getattr(cls, util_name)(normalize, *args, **sub_kwargs[0])
```

Comment:
Why change the function signature?  If normalize is always a keyword argument can just organize it that way?
```suggestion
                    return getattr(cls, util_name)(*args, normalize=normalize, **sub_kwargs[0])
```

Reply:
included suggestion